### ISSUE TITLE
Differentiate dependencies by source

### ISSUE THREAD

**Issue by sanmai-NL** [kbknapp/cargo-graph#37]
To explore the structure of larger projects, e.g. in workspaces, it is helpful to be able to visually distinguish ‘external’ dependencies (e.g. crates.io crates) and in-house dependencies. There seems to be no feature to do this in v0.3.0.

**Comment by przygienda** [comment]
agreed, struggling with same problem, you don't even want to see my graph ;-)

simplest would be something taking a filter regex file for packages e.g. and option for anything that is not in Cargo.toml (pulled in by packages in Cargo.toml indirectly) to be excluded.

Tool is great.

**Comment by Boscop** [comment]
It should be possible to only include crates in the generated graph that are members of the current workspace. 
Is it already possible? 
If not, a flag for this should be added.
